packages/plugin-ext/src/hosted/node/plugin-service.ts

```diff
@@ -113,4 +119,35 @@ export class HostedPluginServerImpl implements HostedPluginServer {
     getExtPluginAPI(): Promise<ExtPluginApi[]> {
         return Promise.resolve(this.extPluginAPIContributions.getContributions().map(p => p.provideApi()));
     }
+
+    protected async localizePlugin(plugin: DeployedPlugin, locale: string): Promise<DeployedPlugin> {
+        plugin = { ...plugin };
```

Comment:
No it doesn't. It also not necessary to copy the element at all, since even if we modify the original object we will re-modify it, as soon as we reload it. Though it may be a code smell to modify it at all.